src/c64/mmu.h

```diff
@@ -128,7 +131,7 @@ class MMU final : public PLA, public sidmemory
      * @param addr the address where to read from
      * @return value at address
      */
-    uint8_t cpuRead(uint_least16_t addr) const { return cpuReadMap[addr >> 12]->peek(addr); }
+    uint8_t cpuRead(uint_least16_t addr) { return (cpuReadMap[addr >> 12])(*this, addr); }
```

Comment:
Again the const, not that it makes a big difference but no point in removing it.

Reply:
Static read functions cannot accept MMU via const reference. Making Bank::peek const also had no luck due to a lot of mutable implementations.